### PR TITLE
Update faq.adoc to include how to dual install a new and old game ver…

### PR DESCRIPTION
**Pull request by charlie504945**
…sion, as well as removing uneccesary steps from the downgrading instructions

Hi,
As it says above, I tweaked the downgrading instructions to remove making a backup of the game files. Since the saves are stored seperatly, deleting the game files manually wont cause any data loss beyond the mod configs at worst. 

I also added instructions on how to have two installations of the game, one older version with mods and the steam installation on the latest update. Technically, this also counts as "how to pirate" if someone gives that copy to another person. HOWEVER, I would like to note that using a copied version you made yourself is not a crime, and that copy and pasting the game files is the first thing anyone would think of when attempting to pirate the game.

Please feel free to contact me with any questions, suggestions or anything at all!

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -617,8 +620,58 @@ In order to launch the game, simply launch it through Steam or the mod manager a
...
+
+3. Create a shortcut for the `.exe` file in the game's root directory, usually `FactoryGameSteam.exe`.
+One way to do this is to right click on it and select create shortcut.
+
+4. Open the shortcut's properties from the right click menu.
+Under the "Shortcut" settings tab, modify the "Target" line to end in ` -NoSteamClient -EpicPortal`.
+Make sure you have a space between the file path and this launch argument.
+
+5. You now have a standalone version of the game that will not update or talk to Steam.
+This means you wont be able to play online with anyone through this version.
```
Comment (Th3Fanbus) [inline modules/ROOT/pages/faq.adoc:659]: typo: won't

**Review by Th3Fanbus — APPROVED** [review]
Looks reasonable, but haven't tested these steps myself

**Review by jsutyla-sml — APPROVED** [review]
Looks good.  I am testing this so I can run my modded game as well as experiment with the 1.0 changes.

**Comment by charlie504945** [comment]
> Looks good. I am testing this so I can run my modded game as well as experiment with the 1.0 changes.

let me know if you run into any issues!

**Pull request merged**